src/raymath.h

```diff
@@ -1161,16 +1160,16 @@ RMAPI Matrix MatrixTranspose(Matrix mat)
 {
     Matrix result = { 0 };
 
-    result.m0 = mat.m0;
-    result.m1 = mat.m4;
-    result.m2 = mat.m8;
-    result.m3 = mat.m12;
-    result.m4 = mat.m1;
-    result.m5 = mat.m5;
-    result.m6 = mat.m9;
-    result.m7 = mat.m13;
-    result.m8 = mat.m2;
-    result.m9 = mat.m6;
+    result.m0  = mat.m0;
```

Comment:
I prefer to avoid this kind of alignments based on additional spaces. One space is ok despite lines are not aligned.